mypy.ini

```diff
@@ -7,6 +7,10 @@ pretty = True
 no_implicit_reexport = True
 no_implicit_optional = True
 
+# overrides for external code
+[mypy-selenium.webdriver.support.ui]
+no_implicit_reexport = False
+
```

Comment:
Is the right answer here to import for the original place instead of doing the import of the implicit reimport here? This feels like it is just covering up a Galaxy bug (importing from the wrong place) - I could be wrong and the Selenium docs or something could point at a implicit re-export but I would be against it.

Reply:
Fixed by direct import (+1: it's better than implicit re-export)